http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/ClusterHelper.scala

```diff
@@ -127,6 +119,74 @@ class ClusterHelper(
         IO.fromFuture(IO(googleIamDAO.removeServiceAccountKey(googleProject, email, keyId)))
     } getOrElse IO.unit
 
+  def createDataprocImageUserGoogleGroupIfItDoesntExist(googleDirectoryDAO: GoogleDirectoryDAO,
+                                                        dpImageUserGoogleGroupName: String,
+                                                        dpImageUserGoogleGroupEmail: WorkbenchEmail): Future[Unit] = {
```

Comment:
I've been trying to standardize on `IO` in this class (and in my other PR, `LeonardoService`). Let's try to use IO everywhere instead of Future going forward.